pkg/vulnsrc/vulnerability/vulnerability_test.go

```diff
@@ -0,0 +1,55 @@
+package vulnerability
+
+import (
+	"testing"
+
+	"github.com/aquasecurity/trivy/pkg/vulnsrc/vulnerability"
+
+	"github.com/aquasecurity/trivy-db/pkg/types"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetDetail(t *testing.T) {
+
+	oldGetVulnerabilityDetailFunc := getVulnerabilityDetailFunc
+	defer func() {
+		getVulnerabilityDetailFunc = oldGetVulnerabilityDetailFunc
+	}()
+
+	getVulnerabilityDetailFunc = func(cveID string) (m map[string]types.VulnerabilityDetail, err error) {
+		return map[string]types.VulnerabilityDetail{
+			vulnerability.RedHat: {
+				ID:          "CVE-2020-1234",
+				CvssScore:   4.3,
+				CvssScoreV3: 5.6,
+				Severity:    types.SeverityHigh,
+				SeverityV3:  types.SeverityCritical,
+				Title:       "test vulnerability",
+				Description: "a test vulnerability where vendor rates it lower than NVD",
+			},
```

Comment:
Recently, Red Hat shows only CVSSv3.
https://access.redhat.com/security/cve/cve-2020-8597

So, we should add the case which doesn't have CvssScore/Severity.

```
			vulnerability.RedHat: {
				ID:          "CVE-2020-1234",
				CvssScoreV3: 5.6,
				SeverityV3:  types.SeverityCritical,
				Title:       "test vulnerability",
				Description: "a test vulnerability where vendor rates it lower than NVD",
			},
```

Reply:
Added a few more tests.